### PR TITLE
Eliminate extra condition block

### PR DESCRIPTION
**Pull request by beoliveira**

### CODE REVIEW

File: React/Base/RCTKeyCommands.m [hunk 1]
```diff
@@ -97,12 +97,11 @@ - (void)RCT_handleKeyCommand:(UIKeyCommand *)key
   static NSTimeInterval lastCommand = 0;
   if (RCTIsIOS8OrEarlier() || CACurrentMediaTime() - lastCommand > 0.5) {
     for (RCTKeyCommand *command in [RCTKeyCommands sharedInstance].commands) {
-      if ([command.keyCommand.input isEqualToString:key.input] &&
-          command.keyCommand.modifierFlags == key.modifierFlags) {
-        if (command.block) {
+      if (([command.keyCommand.input isEqualToString:key.input] &&
+          command.keyCommand.modifierFlags == key.modifierFlags) &&
+		  command.block) {
```
Comment (edvinerikson) [inline React/Base/RCTKeyCommands.m]: Can you replace the tabs with spaces?

**Comment by chirag04** [comment]
@nicklockwood can you review?

File: React/Base/RCTKeyCommands.m [hunk 2]
```diff
@@ -97,12 +97,11 @@ - (void)RCT_handleKeyCommand:(UIKeyCommand *)key
   static NSTimeInterval lastCommand = 0;
   if (RCTIsIOS8OrEarlier() || CACurrentMediaTime() - lastCommand > 0.5) {
     for (RCTKeyCommand *command in [RCTKeyCommands sharedInstance].commands) {
-      if ([command.keyCommand.input isEqualToString:key.input] &&
-          command.keyCommand.modifierFlags == key.modifierFlags) {
-        if (command.block) {
+      if (([command.keyCommand.input isEqualToString:key.input] &&
```
Comment (nicklockwood) [inline React/Base/RCTKeyCommands.m]: Why the extra set of ( ) in the if statement?

File: React/Base/RCTKeyCommands.m [hunk 3]
```diff
@@ -97,12 +97,11 @@ - (void)RCT_handleKeyCommand:(UIKeyCommand *)key
...
+          command.keyCommand.modifierFlags == key.modifierFlags) &&
+      command.block) {
```
Comment (nicklockwood) [inline React/Base/RCTKeyCommands.m]: indenting looks off

**Comment by facebook-github-bot-1** [comment]
@nicklockwood would you mind taking a look at this pull request? It's been a while since the last commit was reviewed.

**Comment by mkonicek** [comment]
Hey @beoliveira! Thanks for making the pull request, but we are closing it due to inactivity (see Nick's comment from November) to make sure all pull requests are either being worked on or closed. If you want to get your proposed changes merged, please rebase your branch with master and send a new pull request :)

**Pull request closed without merging**